src/langs/nl.json

```diff
@@ -0,0 +1,27 @@
+{
+  "alert-secret-not-found": "De gegevens die je zocht bestaan (niet) meer&hellip; - Als je hier informatie verwachtte dan is de link mogelijk al door iemand anders bekeken!",
```

Comment:
Shouldn't the parens be around the "meer"? "bestaan niet / bestaan niet meer"

Reply:
Yes, you're right. I let someone proofread and we both didn't see it. :man_facepalming: